docker/docker-compose.test.yml

```diff
@@ -22,7 +22,6 @@ services:
     build:
       context: ..
       dockerfile: Dockerfile
-    command: py.test
```

Comment:
Output (reports) also needs to be specified here. Jenkins doesn't use test.sh script.